frontend/src/assets/i18n/en.json

```diff
@@ -96,13 +96,14 @@
                 "select-area-first-label": "Start by selecting an area"
             },
             "split-scenario-settings": {
-                "title": "Split scenario by area",
-                "detail": "Generate a new scenario for each included area",
+                "title": "Multiple area actions",
+                "detail": "Generate a new scenario for each included area, or run a batch calculation on the included areas individually. Note that the latter action will not generate new scenarios.",
```

Comment:
Ta bort kommatecken.
`"detail": "Generate a new scenario for each included area or run a batch calculation on the included areas individually. Note that the latter action will not generate new scenarios.",`